pkg/madmin/README.md

```diff
@@ -537,3 +537,22 @@ __Example__
 
     log.Println("Profiling data successfully downloaded.")
 ```
+
+<a name="ListenTrace"></a>
+### ListenTrace(allTrace bool,doneCh <-chan struct{}) <-chan TraceInfo
+get http trace from minio server
```

Comment:
We should explain a bit more here, that how it is that trace comes from all servers in a cluster etc. Start the sentence with a capitalization since this is a document.